CONTRIBUTING.md

```diff
@@ -99,7 +99,7 @@ Follow GitHub's fork-branch-pull request pattern.
 ## Writing High-Quality Code
 
 1. Follow cFS code conventions (formatting, symbol naming, file naming, etc). Do *not* change/reformat existing code, except to address your changes.
-    - The cFS submodules uses the Format Check workflow to ensure users follow the [clang-format-10](https://releases.llvm.org/10.0.0/tools/clang/docs/ClangFormat.html) style. 
+    - The cFS submodules uses the Format Check workflow to ensure users follow the [clang-format-10](https://releases.llvm.org/10.0.0/tools/clang/docs/ClangFormat.html) style. For more information on how to use the Format Check workflow, view [Using GitHub Actions Workflows](#using-github-actions-workflows).  
```

Comment:
I would point to the .clang-format file under instead of the clang format tool repo.

Reply:
Changed link from https://releases.llvm.org/10.0.0/tools/clang/docs/ClangFormat.html to https://github.com/nasa/cFS/blob/main/.clang-format.